### ISSUE TITLE
train asian face dataset with pre-trained model

### ISSUE THREAD

**Issue by buptljl** [davidsandberg/facenet#936]
pre-trained model has poor results on asian face, I want to train asian face dataset with pre-trained model. Is this available？

**Comment by shoyo-su** [comment]
Yes that is available you just have to change the last layer of the network with labels of asian faces .Rest the network would take care of itself and readjust all its values accordingly.Try coding this and if you still face problems feel free to reach out!!!!

**Comment by RAJA-PARIKSHAT** [comment]
@ajitcool19 Can you tell how would you do it, if we want to train pretrained model with Asian faces, I cant figure out how to train if I don't want to concretely define the output classes.
